### ISSUE TITLE
Support minimal derivatives from fMRIPrep

### ISSUE THREAD

**Issue by tsalo** [nipreps/fmripost-aroma#12]
### What would you like to see added in this software?

This came up in today's NiPreps roundup call. I initially assumed that the fMRIPost workflow would use standard-space derivatives from fMRIPrep. However, what would be better would be for the fMRIPost Apps to have workflows that apply transforms from the minimal fMRIPrep outputs.

### Do you have any interest in helping implement the feature?

Yes

**Comment by effigies** [comment]
Yes.

**Comment by tsalo** [comment]
Since derivatives datasets can also be self-contained datasets, I think providing just derivatives (assuming they were run with "full" mode) should be allowed as well. Not sure how well all of that would play with the BIDS execution spec though.

**Comment by effigies** [comment]
Yeah, that should be doable. Look for space-... And only look up the bits to recalculate it if needed.

**Comment by tsalo** [comment]
With minimal mode, there won't be a confounds timeseries file, so in that case:

1. `--dummy-scans` needs to be required.
2. Motion parameters need to be extracted from the motion correction transform file.
